vmdb/app/views/shared/summary/_textual_multivalue.html.haml

```diff
@@ -0,0 +1,7 @@
+%table{:cellpadding => "0", :cellspacing => "0"}
+  - items.each do |item|
+    %tr{:class => item[:title] ? 'no-hover' : '', :title => item[:title] ? item[:title] && item[:link] : "", :onclick => item[:link] ? "DoNav('#{item[:link]}');" : ""}
```

Comment:
here you output an empty class for the else branche, above you have `false`

these should be consistent